doc/getting-started/getting-started.rst

```diff
@@ -378,121 +389,155 @@ their attributes, and the resources they have access to.
-         :class: drop-shadow
+      cd ~/acrn-work/acrn-hypervisor
+      make configurator
 
-   #. In the dialog box, select **shared_launch_6user_vm** as the default launch
-      setting and click **OK**. Because our sample ``shared`` scenario defines
-      six User VMs, we're using this ``shared_launch_6user_vm`` launch XML
-      configuration.
+#. Install the ACRN Configurator:
 
-      .. image:: ./images/gsg_config_launch_load.png
-         :class: drop-shadow
+   .. code-block:: bash
+
+      sudo apt install -y ~/acrn-work/acrn-hypervisor/build/acrn-configurator_*_amd64.deb
 
-      Of the six User VMs, we will use User VM 3 and modify its default settings to run Ubuntu 20.04.
+#. Launch the ACRN Configurator:
 
-   #. Scroll down, find **User VM 3**, and change the **mem_size** to **1024**.
-      Ubuntu 20.04 needs at least 1024 megabytes of memory to boot.
+   .. code-block:: bash
 
-      .. image:: ./images/gsg_config_mem.png
-         :class: drop-shadow
+      acrn-configurator
 
-   #. Under virtio_devices, change the **block** to
-      **/home/acrn/acrn-work/ubuntu-20.04.4-desktop-amd64.iso**. The parameter
-      specifies the VM's OS image and its location on the target system. Later
-      in this guide, you will save the ISO file to that directory.
+#. On the left-hand side, look for **Start a new configuration**.
+   Confirm that the working folder is ``<path to>/acrn-work/MyConfiguration``. Click **Use This Folder**.
+
+   TODO add screenshots to Configurator steps
+
+#. Import your board configuration file as follows:
+
+   a. Under **Import a board configuration file**, click **Browse for file**.
+
+   #. Browse to ``~/acrn-work/my_board.xml`` and click **Open**.
+
+   #. Click **Import Board File**.
+
+   The ACRN Configurator makes a copy of your board file, changes the file
+   extension to ``.board.xml``, and saves the file to the working folder.
 
-      .. image:: ./images/gsg_config_blk.png
-         :class: drop-shadow
+#. Create a new scenario as follows:
 
-   #. Click the **Generate Launch Script** button.
+   a. Under **Create new or import an existing scenario**, click **Create
+      Scenario**.
 
-      .. image:: ./images/gsg_config_launch_generate.png
-         :class: drop-shadow
+   #. In the dialog box, confirm that **Shared (Post-launched VMs only)** is
+      selected.
 
-   #. In the dialog box, type ``/home/<username>/acrn-work/`` in the Launch XML
-      Path field. In the following example, ``acrn`` is the username. Click
-      **Submit** to save the script.
+   #. Confirm that one Service VM and one post-launched VM are selected.
 
-      .. image:: ./images/gsg_config_launch_save.png
-         :class: drop-shadow
+   #. Click **Create**.
 
-#. Close the browser and press :kbd:`CTRL` + :kbd:`C` to terminate the
-   ``acrn_configurator.py`` program running in the terminal window.
+#. Generate the scenario configuration file and launch script:
```

Comment:
```suggestion
#. Generate the scenario configuration file and launch scripts:
```
? Or is there just one launch script to be generated?

Reply:
Just one script (one post-launched VM) in this new GSG